contracts/Core/StakeManager.sol

```diff
@@ -103,21 +102,22 @@ contract StakeManager is Initializable, ACL, StakeStorage, StateManager, Pause {
         uint256 amount
     ) external initialized checkEpochAndState(State.Commit, epoch, parameters.epochLength()) whenNotPaused {
         require(stakers[stakerId].acceptDelegation, "Delegetion not accpected");
-        // Step 1:  Razor Token Transfer : Amount
-        require(razor.transferFrom(msg.sender, address(this), amount), "RZR token transfer failed");
+        require(stakers[stakerId].tokenAddress != address(0x0), "Staker has not staked yet");
```

Comment:
this require was removed from master, please remove it from here as well, CC @GauravJain9 PR